questionary/prompts/checkbox.py

```diff
@@ -246,6 +246,30 @@ def all(_event):
 
         perform_validation(get_selected_values())
 
+    @bindings.add(Keys.Home, eager=True)
+    def move_cursor_start(event):
+        ic.pointed_at = 0
+
+    @bindings.add(Keys.End, eager=True)
+    def move_cursor_end(event):
+        ic.pointed_at = len(ic.choices) - 1
+
+    @bindings.add(Keys.PageDown, eager=True)
+    def move_cursor_down_fast(event):
+        for _ in range(10):
+            ic.select_next()
+
+        while not ic.is_selection_valid():
```

Comment:
What happens if you have a list with 1 valid item and 99 invalid items? In such a case, I think this might cause an infinite loop (unless `ic.select_next()` loops around to the start of the list)? Can you write a test to cover this.